Maths/ZellersCongruence.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+"""Zellers Congruence Birthday Algorithm
+    Find out what day of the week you were born on, or any other date. 
+"""
+
+def zeller(date_input):
+
```

Comment:
Please consider adding some doctests here.

Reply:
Resolving and resubmitting. Thanks